backend/python/app/__init__.py

```diff
@@ -32,22 +33,29 @@ def create_app(config_name):
     )
 
     app = Flask(__name__)
+    mysql = MySQL()
     app.config.from_object(app_config[config_name])
 
     app.config["CORS_ORIGINS"] = ["http://localhost:3000"]
     app.config["CORS_SUPPORTS_CREDENTIALS"] = True
     CORS(app)
 
+    app.config["MYSQL_HOST"] = os.getenv("MYSQL_HOST")
```

Comment:
I don't believe these are necessary. I deleted l43-47 and had no issues. Does it impact your set up?